### PR TITLE
chore: updating staking deployment scripts

### PR DESCRIPTION
**Pull request by kupermind**
- Updating staking deployment scripts.

### CODE REVIEW

**Review by 77ph — APPROVED** [review]

File: scripts/deployment/l2/globals_base_mainnet.json [hunk 1]
```diff
@@ -1 +1 @@
...
...enUtilityAddress":"0x34C895f302D0b5cf52ec0Edd3945321EB0f83dd5","serviceManagerTokenAddress":"0x63e66d7ad413C01A7b49C7FF4e3Bb765C4E4bd1b","gnosisSafeMultisigImplementationAddress":"0xBb7e1D6Cb6F243D6bdE81CE92a9f2aFF7Fbe7eac","gnosisSafeSameAddressMultisigImplementationAddress":"0xFbBEc0C8b13B38a9aC0499694A69a10204c5E2aB","stakingTokenAddress":"","stakingNativeTokenAddress":"","minStakingDepositLimit":"1000000000000000000000","timeForEmissionsLimit":"2592000","numServicesLimit":"100","apyLimit":"3000000000000000000","stakingVerifierAddress":"","stakingFactoryAddress":"","livenessRatio":"700000000000000","stakingActivityCheckerAddress":"","stakingParams":{"metadataHash":"","maxNumServices":"100","rewardsPerSecond":"1000000000000000","minStakingDeposit":"50000000000000000000","minNumStakingPeriods":"3","maxNumInactivityPeriods":"3","livenessPeriod":"86400","timeForEmissions":"2592000","numAgentInstances":"1","agentIds":["14"],"threshold":"0","configHash":"0x0000000000000000000000000000000000000000000000000000000000000000","proxyHash":"0xb89c1b3bdf2cf8827818646bce9a8f6e372885f8c55e5c07acbd307cb133b000","serviceRegistry":"0x3C1fF68f5aa342D296d4DEe4Bb1cACCA912D95fE","activityChecker":""}}
```
Comment (DavidMinarsch) [inline scripts/deployment/l2/globals_base_mainnet.json:1]: bridgeMediatorAddress also missing?

Comment (DavidMinarsch) [reply scripts/deployment/l2/globals_base_mainnet.json:1]: same for celo

File: scripts/deployment/l2/globals_ethereum_mainnet.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (DavidMinarsch) [inline scripts/deployment/l2/globals_ethereum_mainnet.json:1]: wrong here

File: scripts/deployment/l2/globals_base_mainnet.json (hunk 1 above)
Comment (DavidMinarsch) [reply scripts/deployment/l2/globals_base_mainnet.json:1]: same for optimism

Comment (DavidMinarsch) [reply scripts/deployment/l2/globals_base_mainnet.json:1]: https://basescan.org/address/0xEB5638eefE289691EcE01943f768EDBF96258a80

Comment (kupermind) [reply scripts/deployment/l2/globals_base_mainnet.json:1]: `bridgeMediatorAddress` is irrelevant now

File: scripts/deployment/l2/globals_ethereum_mainnet.json (hunk 2 above)
Comment (kupermind) [reply scripts/deployment/l2/globals_ethereum_mainnet.json:1]: let's keep it here for now

Comment (kupermind) [reply scripts/deployment/l2/globals_ethereum_mainnet.json:1]: I'll merge it back to the main one later

File: scripts/deployment/l2/globals_base_mainnet.json (hunk 1 above)
Comment (DavidMinarsch) [reply scripts/deployment/l2/globals_base_mainnet.json:1]: https://basescan.org/address/0x10c5525F77F13b28f42c5626240c001c2D57CAd4

Comment (DavidMinarsch) [reply scripts/deployment/l2/globals_base_mainnet.json:1]: https://basescan.org/address/0x1cEe30D08943EB58EFF84DD1AB44a6ee6FEff63a

Comment (DavidMinarsch) [reply scripts/deployment/l2/globals_base_mainnet.json:1]: https://polygonscan.com/address/0x4aba1Cf7a39a51D75cBa789f5f21cf4882162519

Comment (DavidMinarsch) [reply scripts/deployment/l2/globals_base_mainnet.json:1]: https://polygonscan.com/address/0x8Bc0a5c5B1612A16983B01ecd7ae130E728390CB

Comment (DavidMinarsch) [reply scripts/deployment/l2/globals_base_mainnet.json:1]: https://polygonscan.com/address/0x46C0D07F55d4F9B5Eed2Fc9680B5953e5fd7b461

Comment (DavidMinarsch) [reply scripts/deployment/l2/globals_base_mainnet.json:1]: https://celoscan.io/address/0xe1E1B286EbE95b39F785d8069f2248ae9C41b7a9

Comment (DavidMinarsch) [reply scripts/deployment/l2/globals_base_mainnet.json:1]: https://celoscan.io/address/0xc40C79C275F3fA1F3f4c723755C81ED2D53A8D81

Comment (DavidMinarsch) [reply scripts/deployment/l2/globals_base_mainnet.json:1]: https://celoscan.io/address/0x1c2cD884127b080F940b7546c1e9aaf525b1FA55

Comment (DavidMinarsch) [reply scripts/deployment/l2/globals_base_mainnet.json:1]: https://etherscan.io/address/0x0Dc23eEf3bC64CF3cbd8f9329B57AE4C4f28d5d2

Comment (DavidMinarsch) [reply scripts/deployment/l2/globals_base_mainnet.json:1]: https://etherscan.io/address/0x4503b79d468e81Ad2d4bd6Db991810da269bA777

Comment (DavidMinarsch) [reply scripts/deployment/l2/globals_base_mainnet.json:1]: https://etherscan.io/address/0xEBdde456EA288b49f7D5975E7659bA1Ccf607efc

**Review by DavidMinarsch — APPROVED** [review]

**Review by 77ph — APPROVED** [review]

**Pull request merged**